crates/extension_api/wit/since_v0.4.0/extension.wit

```diff
@@ -95,6 +95,9 @@ world extension {
     /// Returns the workspace configuration options to pass to the language server.
     export language-server-workspace-configuration: func(language-server-id: string, worktree: borrow<worktree>) -> result<option<string>, string>;
 
+    /// Returns the workspace configuration options to pass to the other language server.
+    export additional-language-server-workspace-configuration: func(language-server-id: string, worktree: borrow<worktree>) -> result<option<string>, string>;
```

Comment:
Isn't it possible for this call to be ambiguous, as the implementer does not know which language server the additional configuration will be applied to?

In order for this API to be clear, I think we also need to be passing the language server ID of the _other_ language server that we're providing configuration for.

Reply:
Sure, right now, the root keys of the JSON act like other language server IDs. But, providing another language server ID as a parameter is makes API more clear, and then we can simply use match.